discojs/package.json

```diff
@@ -16,7 +14,11 @@
   },
   "homepage": "https://github.com/epfml/disco#readme",
   "dependencies": {
-    "@epfml/discojs-core": "file:../discojs-core/dist/browser",
+    "@epfml/discojs-node": "file:./discojs-node",
```

Comment:
Can we move this to devDependencies? If it's used only in test it doesn't need to be here